include/fastdds/rtps/security/cryptography/Cryptography.h

```diff
@@ -44,11 +46,19 @@ class Cryptography
 
     CryptoTransform* cryptotransform() { return m_cryptotransform; }
 
+    bool set_logger(Logging* logger, SecurityException& /*exception*/) { m_logger = logger; return true; }
+
 protected:
 
+    const Logging* get_logger() { return m_logger; }
+
     CryptoKeyExchange *m_cryptokeyexchange;
     CryptoKeyFactory *m_cryptokeyfactory;
     CryptoTransform *m_cryptotransform;
+
+private:
+
+    Logging *m_logger;
```

Comment:
Why not `= nullptr` like the others?

Reply:
which `others`? Other pointers in this class are not ` = nullptr`. Are you mistaken with another class maybe?

Reply:
Mistaken? Me? Heaven's no! I was referring to e.g. [this](https://github.com/canonical/Fast-RTPS/pull/1/files/aafbba0eba453b6834d39b60b0a958d383a83de7#diff-3efc66f9a9acbebd29eb69bbee7b0328R238) where you just set the logger to nullptr here instead of in the constructor.

Reply:
I agree with you and would prefer to `= nullptr` at the declaration site but other pointers in this particular plugin class do not do that thus I complain to the local style.

Reply:
Ah ha, fair enough.